model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -337,28 +345,86 @@ def _create_default_run_config(self) -> RunConfig:
         default_run_config = RunConfig(self._triton_env)
 
         for model in self._models:
-            default_model_config = BaseModelConfigGenerator.make_model_config(
-                param_combo={},
-                model=model,
-                model_variant_name_manager=self._model_variant_name_manager)
+            if model.model_name() in self._ensemble_submodels:
+                default_run_config.add_model_run_config(
+                    self._create_default_ensemble_model_run_config(model))
+            else:
+                default_run_config.add_model_run_config(
+                    self._create_default_model_run_config(model))
 
-            default_perf_analyzer_config = PerfAnalyzerConfig()
-            default_perf_analyzer_config.update_config_from_profile_config(
-                default_model_config.get_field('name'), self._config)
+        return default_run_config
 
-            perf_config_params = {'batch-size': 1, 'concurrency-range': 1}
-            default_perf_analyzer_config.update_config(perf_config_params)
+    def _create_default_ensemble_model_run_config(
+            self, model: ModelProfileSpec) -> ModelRunConfig:
+        default_submodel_configs = self._create_default_submodel_configs(model)
 
-            default_perf_analyzer_config.update_config(
-                model.perf_analyzer_flags())
+        default_ensemble_model_config = BaseModelConfigGenerator.make_ensemble_model_config(
+            model=model,
+            ensemble_submodels=default_submodel_configs,
+            model_variant_name_manager=self._model_variant_name_manager)
 
-            default_model_run_config = ModelRunConfig(
-                model.model_name(), default_model_config,
-                default_perf_analyzer_config)
+        default_perf_analyzer_config = self._create_default_perf_analyzer_config(
+            model, default_ensemble_model_config)
 
-            default_run_config.add_model_run_config(default_model_run_config)
+        default_model_run_config = ModelRunConfig(
+            model.model_name(), default_ensemble_model_config,
+            default_perf_analyzer_config)
 
-        return default_run_config
+        default_model_run_config = self._add_submodels_to_model_run_config(
+            default_model_run_config, default_submodel_configs)
+
+        return default_model_run_config
+
+    def _create_default_submodel_configs(
+            self, model: ModelProfileSpec) -> List[ModelConfig]:
+        default_submodel_configs: List[ModelConfig] = []
+        for submodel in self._ensemble_submodels[model.model_name()]:
```

Comment:
we should assert somewhere if we are ever ensemble with no submodels

Reply:
Maybe, but it doesn't go here. This logic works fine even if no submodels exists, it will just return an empty list. 
How much responsibility should MA have to ensure that the user forms a correct config.pbtxt?

Reply:
>How much responsibility should MA have to ensure that the user forms a correct config.pbtxt?
None, but we should still protect our code to not take bad paths.  I agree that it shouldn't go there. Just mentioning it should happen somewhere.